requre/helpers/requests_response.py

```diff
@@ -174,7 +179,8 @@ def decorator(
         map_function_to_item=None,
         storage_object_kwargs=None,
         cassette: Cassette = None,
-        response_headers_to_drop=None
+        response_headers_to_drop=None,
+        stack_internal_check=True,
```

Comment:
I would describe this in the docstring and maybe pick a better name...

Reply:
sure, I'll describe it docstring